app/schemas/models/user.coffee

```diff
@@ -346,6 +346,16 @@ _.extend UserSchema.properties,
   role: {type: 'string', enum: ["God", "advisor", "parent", "principal", "student", "superintendent", "teacher", "technology coordinator"]}
   birthday: c.stringDate({title: "Birthday"})
   lastAchievementChecked: c.stringDate({ name: 'Last Achievement Checked' })
+  signupQueryParams: { type: 'object', properties: {
+    il_id: {type: 'string'}
+    first_name: {type: 'string'}
```

Comment:
To add some permanence to the values and record exactly what was provided on signup, so they can be referenced. The attributes are underscored because... I believe that's what the originally were. But they should probably be changed.